app/javascript/components/Admin/Purchases/ResendReceiptForm.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
```

Comment:
I think it still makes sense to move this component out of the `server-components` directory since it is not going to be rendered via React on Rails at all when we drop the old admin UI.